packages/components/src/internal/components/base/LabelHelpTip.tsx

```diff
@@ -44,7 +37,5 @@ export const LabelHelpTip: FC<Props> = memo(props => {
 LabelHelpTip.displayName = 'LabelHelpTip';
 
 LabelHelpTip.defaultProps = {
-    customStyle: {},
```

Comment:
I'm curious what the usages were that were passing in `customStyle` and why? was it just to change the color or something like that?

Reply:
CollapsiblePanelHeader is the only one I can find

Reply:
Yes, and with the styles it was passing it looked really bad, alignment was way off. By removing the custom style it looks much better.